### ISSUE TITLE
Some questions

### ISSUE THREAD

**Issue by xgzng** [farzad-bz/DeCo-Diff#6]
Hello, this is a great job. I have a few questions:
1. The minimum gpu memory size required for model training and the model training time after reducing the batchsize
2. What is the fps size during model inference

**Comment by farzad-bz** [comment]
Hello, and thank you for your interest!

GPU Memory Usage:

- For the Large model:
> Batch size 128: ~48 GB
> Batch size 64: ~26 GB

- For the XS model:
> Batch size 64: ~17 GB
> Batch size 32: ~10 GB

If you're using smaller batch sizes, I recommend increasing the gradient accumulation steps and possibly the number of training epochs to maintain effective learning.

Inference Speed for the Large model:

> Batch size 8: FPS is approximately 5.5
> Batch size 64: FPS is approximately 10

Let me know if you have any other questions!

**Comment by xgzng** [comment]
Thank you very much for your prompt response.  I don't have any further questions at the moment, so I will close this issue.
